config/locales/diaspora/en.yml

```diff
@@ -812,6 +812,27 @@ en:
         Cheers,
 
         The diaspora* email robot!
+    export_photos_email:
+      subject: "Your photos are ready for download, %{name}"
+      body: |-
+        Hello %{name},
+
+        Your photos have been processed and are ready for download by following [this link](%{url}).
+
+        Cheers,
+
+        The diaspora* email robot!
+    export_photos_failure_email:
+      subject: "We’re sorry, there was an issue with your photos, %{name}"
+      body: |-
+        Hello %{name}
+
+        We’ve encountered an issue while processing your photos for download.
+        Please try again!
+
+        Cheers,
```

Comment:
A purely style comment: I think saying 'cheers' here might seem a bit dismissive, because it's giving bad news, and 'cheers' is more of an upbeat/happy sign-off. Perhaps remove 'we're sorry' from the email title and change this line from 'Cheers,' to 'Sorry,'.